Makefile

```diff
@@ -1,7 +1,12 @@
+VERSION = "unset"
+DATE=$(shell date -u +%Y-%m-%d-%H:%M:%S-%Z)
 .PHONY: build
 build:
 	cd cmd/proxy && buffalo build
 
+build-ver:
+	GO111MODULE=on CGO_ENABLED=0 go build -mod=vendor -ldflags "-X github.com/gomods/athens/pkg/build.version=$(VERSION) -X github.com/gomods/athens/pkg/build.buildDate=$(DATE)" -o athens ./cmd/proxy
```

Comment:
Would this not allow users to create random build version?

For example,

can't a user do this 
`make build-ver VERSION=0.3.0`

when it is actually `master` or `VERSION=0.2.0`

Reply:
Yes, it's up to the user but I thought that is the purpose of #887 I wasn't on the meeting though...

Reply:
Oh! I did not go through the ticket and forgot about the purpose. It Looks good to me. I will merge it.